components/TheFooter.vue

```diff
@@ -1,15 +1,154 @@
 <template>
-  <div>
-    <div class="container">new footer</div>
-  </div>
+  <footer class="footer-container section">
+    <div
+      class="container is-flex is-align-items-start is-justify-content-space-between">
+      <div class="footer-container-subs is-flex is-flex-direction-column">
+        <span class="footer-container-heading">{{
```

Comment:
instead of span I would use something like `<h4 class="has-text-white subtitle is-5">` that would do the same without your css

Reply:
semantic + respect spacing from design